### DISCUSSION TITLE
L1 Cache could be described better.

### DISCUSSION THREAD

**Discussion by SkybuckFlying** [turkaysoft/glow#43] [Ideas]
In the sense of L1 Instruction Cache Size, L1 Data Cache Size.

And also cache size per processor, instead of "total", which is a bit misleading...

<------- PROCESSOR INFORMATION ------->

Processor Model: AMD Ryzen 9 7950X3D 16-Core Processor
Processor Manufacturer: AuthenticAMD
Processor Architecture: 64 Bit - (x64)
Processor Speed: 4.20 GHz
Default Processor Speed: 4.20 GHz
L1 Cache Size: 1 MB
L2 Cache Size: 16 MB
L3 Cache Size: 128 MB
Core Count: 16
Logical Core Count: 32
Processor Usage: 1.31%
Processor Process Count: 230
Socket Description: AM5
Processor Family: AMD X64 Family: 25 Model: 97 Phase: 2
Virtualization: Active
Unique Processor ID: 178BFBFF00A60F12
Processor Code Sets: 3DNOW! Extended, ABM, AES, EM64T, MMX, NODEID_MSR, SSE, SSE4.1, SSE4.2, SSE4a, SSE5

**Comment by turkaysoft** [comment]
> In the sense of L1 Instruction Cache Size, L1 Data Cache Size.
> 
> And also cache size per processor, instead of "total", which is a bit misleading...
> 
> <------- PROCESSOR INFORMATION ------->
> 
> Processor Model: AMD Ryzen 9 7950X3D 16-Core Processor Processor Manufacturer: AuthenticAMD Processor Architecture: 64 Bit - (x64) Processor Speed: 4.20 GHz Default Processor Speed: 4.20 GHz L1 Cache Size: 1 MB L2 Cache Size: 16 MB L3 Cache Size: 128 MB Core Count: 16 Logical Core Count: 32 Processor Usage: 1.31% Processor Process Count: 230 Socket Description: AM5 Processor Family: AMD X64 Family: 25 Model: 97 Phase: 2 Virtualization: Active Unique Processor ID: 178BFBFF00A60F12 Processor Code Sets: 3DNOW! Extended, ABM, AES, EM64T, MMX, NODEID_MSR, SSE, SSE4.1, SSE4.2, SSE4a, SSE5

Thank you for your suggestion.
